### PR TITLE
add and getting daily calories

### PR DESCRIPTION
**Pull request by VishalPhirkoj**

### CODE REVIEW

File: capstone-backend/controllers/dailyCalorie.controller.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const DailyCalorie = require("../models/dailyCalorie.model");
+const dotenv = require("dotenv");
+dotenv.config("../.env");
+
+const addDailyCalorie = async (req, res) => {
+  try {
+    // let currentDate = new Date();
+    console.log(Date());
+    const obj = {
+      Date: Date(),
```
Comment (romario-0) [inline capstone-backend/controllers/dailyCalorie.controller.js]: This date will be obtained from front-end.
Also add a validation to check if the date is <= today's date

File: capstone-backend/controllers/dailyCalorie.controller.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      dailyCalorie: response,
+    });
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const getDailyCalorie = async (req, res) => {
+  try {
+    const getDailyCalories = await DailyCalorie.find({ userId: req.user.id });
```
Comment (romario-0) [inline capstone-backend/controllers/dailyCalorie.controller.js]: Can you add a condition to get values within specific date range

**Review by romario-0 — CHANGES_REQUESTED** [review]

File: capstone-backend/controllers/dailyCalorie.controller.js (hunk 1 above)
Comment (VishalPhirkoj) [reply capstone-backend/controllers/dailyCalorie.controller.js]: Done

File: capstone-backend/controllers/dailyCalorie.controller.js (hunk 2 above)
Comment (VishalPhirkoj) [reply capstone-backend/controllers/dailyCalorie.controller.js]: Done

**Review by romario-0 — APPROVED** [review]

**Pull request merged**